lib/parser.js

```diff
@@ -134,23 +134,24 @@ function parseFromUrl(url, fetchOptions = {}, options) {
 async function dereference(parsedJSON, initialFormat, asyncapiYAMLorJSON, options) {
   try {
     return await refParser.dereference(options.path, parsedJSON, {
+      continueOnError: true,
       parse: options.parse,
       resolve: options.resolve,
       dereference: options.dereference,
     });
   } catch (err) {
     throw new ParserError({
       type: 'dereference-error',
-      title: err.message,
+      title: err.errors[0].message,
```

Comment:
looking at the current structure of the ParserError I just pick up the first message from the list of errors. We might consider a change here in case we see a need in the future. I can create a follow-up issue if you think it is needed, but I'm affraid it will get stale pretty quickly and think it is better to leave it as it is and go back to it once we really need more details here for some UX purposes